javascript/ql/src/semmle/javascript/ApiGraphs.qll

```diff
@@ -35,6 +35,17 @@ module API {
       )
     }
 
+    /**
+     * Gets a source-node corresponding to a use of the API component represented by this node.
+     *
+     * For example, `require('fs').readFileSync` is a use of the function `readFileSync` from the
+     * `fs` module, and `require('fs').readFileSync(file)` is a use of the result of that function.
+     *
+     * As another example, in the assignment `exports.plusOne = (x) => x+1` the two references to
+     * `x` are uses of the first parameter of `plusOne`.
+     */
+    DataFlow::SourceNode getASourceUse() { Impl::use(this, result) }
```

Comment:
I don't like the name. How about something like `getAReference`? Also, you just copy-pasted the doc comment from `getAUse`; can you adjust it to clarify the difference between the two predicates?

Reply:
I found out that I also want to use `Impl::trackUseNode` in this new predicate, so the two predicates are very similar now.   
Your naming suggestion is definitely better than what I came up with.